addons/l10n_id_efaktur/models/efaktur.py

```diff
@@ -14,44 +14,54 @@ class Efaktur(models.Model):
     company_id = fields.Many2one('res.company', required=True, default=lambda self: self.env.company)
     max = fields.Char(compute='_compute_default', store=True, readonly=False)
     min = fields.Char(compute='_compute_default', store=True, readonly=False)
-    available = fields.Integer(compute='_compute_available', store=True)
+    available = fields.Integer(readonly=True)
```

Comment:
There is a flaw in the system: if I increase the max of a range (which I can do), then what? available isn't recomputed for example.